1_processing/1_decide_threshold.ipynb

```diff
@@ -841,147 +841,147 @@
        "  <thead>\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #11.        pao1_sra, color="#C6A9B5", label="PAO1", kde=False, hist_kws={"alpha": 0.8}
</pre><p>What do you think about decreasing the alpha here slightly? It's ok as is, but i think ti could help differentiate the two a bit more. Otherwise, love the log scale, much easier to see!</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/greenelab/core-accessory-interactome/pull/58/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='22'/>